packages/office-ui-fabric-react/src/components/MarqueeSelection/__snapshots__/MarqueeSelection.test.tsx.snap

```diff
@@ -365,7 +364,6 @@ exports[`MarqueeSelection renders MarqueeSelection correctly 1`] = `
             "neutralQuaternary": "#d0d0d0",
             "neutralQuaternaryAlt": "#dadada",
             "neutralSecondary": "#666666",
-            "neutralSecondaryAlt": "#767676",
```

Comment:
This doesn't seem right. None of your changes should have triggered this. Please make sure you do a `rush build` or `npm install` locally to make sure all of your packages are up to date.